### PR TITLE
SQS with separate service name

### PR DESCRIPTION
**Pull request by lpriima**

### CODE REVIEW

File: dd-java-agent/instrumentation/aws-java-sdk-1.11.0/src/main/java/datadog/trace/instrumentation/aws/v0/AwsSdkClientDecorator.java [hunk 1]
```diff
@@ -20,6 +21,7 @@
   private static final Pattern AMAZON_PATTERN = Pattern.compile("Amazon", Pattern.LITERAL);
 
   static final CharSequence COMPONENT_NAME = UTF8BytesString.create("java-aws-sdk");
+  static final String SQS_SERVCIE_NAME = "java-aws-sqs";
```
Comment (devinsba) [inline dd-java-agent/instrumentation/aws-java-sdk-1.11.0/src/main/java/datadog/trace/instrumentation/aws/v0/AwsSdkClientDecorator.java:24]: looks like you've got a typo here

**Comment by devinsba** [comment]
Is there something missing before you think this is ready for review?

**Comment by lpriima** [comment]
> Is there something missing before you think this is ready for review?

still waiting for customer feedback about it and `DEFAULT_AWS_SQS_AS_SEPARATE_SERVICE` should be `false` before it's ready to review

**Comment by mcculls** [comment]
replaced by #3096

**Pull request closed without merging**